src/cli.ts

```diff
@@ -209,9 +213,16 @@ const createPatientBundles = (patientBundleDir: string) => {
  * Measure Reports to file.
  */
 const runMeasureCalculation = async () => {
+  if (options.from !== undefined && !moment(options.from).isValid())
+    console.log(
+      'Date format of --from param : ' + options.from + ' is not valid at ' + moment(options.from).invalidAt()
+    );
+  if (options.to !== undefined && !moment(options.to).isValid())
+    console.log('Date format of --to param : ' + options.to + ' is not valid at ' + moment(options.to).invalidAt());
+
   const calculationOptions: CalculatorTypes.CalculationOptions = {
-    measurementPeriodStart: '2019-01-01',
-    measurementPeriodEnd: '2019-12-31',
+    measurementPeriodStart: options.from === undefined ? '1000-01-01' : options.from,
+    measurementPeriodEnd: options.to === undefined ? '9999-12-31' : options.to,
     reportType: 'summary',
```

Comment:
```suggestion
    reportType: 'summary',
    ...(options.from && { measurementPeriodStart: options.from }),
    ...(options.to && { measurementPeriodEnd: options.to }),
```
Suggestion to only include the custom `measurementPeriodStart` and `measurementPeriodEnd` parameters if they are defined via `options.from` and `options.to`. If these values are not defined, then the measure calculation will use the `effectivePeriod` field defined on the FHIR Measure. Since we have that fallback handling to use the `effectivePeriod`, we do not need to set arbitrary values for the `measurementPeriodStart` or `measurementPeriodEnd`.

Reply:
That makes sense. We don't have to add unnecessarily additional handling unless it is part of requirements.